src/electron/window.main.ts

```diff
@@ -221,4 +228,8 @@ export class WindowMain {
         return state != null && Number.isInteger(state.x) && Number.isInteger(state.y) &&
             Number.isInteger(state.width) && state.width > 0 && Number.isInteger(state.height) && state.height > 0;
     }
+
+    public quit() {
```

Comment:
Is this function necessary? Looks like `isQuitting` is already public.

Reply:
Function removed.